include/Foundation/NSDateComponents.h

```diff
@@ -48,4 +47,6 @@ FOUNDATION_EXPORT_CLASS
 @property (getter=isLeapMonth) BOOL leapMonth;
 - (NSInteger)valueForComponent:(NSCalendarUnit)unit;
 - (void)setValue:(NSInteger)value forComponent:(NSCalendarUnit)unit;
+- (NSInteger)week;
+- (void)setWeek:(NSInteger)v;
```

Comment:
Hoenstly, I'm pretty confused by this. What benefit is there to remove the week property and replacing it with an ivar and some selectors? This does the same thing as having a property for week does it not?